Mathlib/Algebra/Order/Monoid/Unbundled/Pow.lean

```diff
@@ -52,9 +52,9 @@ theorem pow_lt_one_of_lt {a : M} {n : ℕ} (h : a < 1) (hn : n ≠ 0) : a ^ n <
 
 end Left
 
-@[to_additive nsmul_nonneg] alias one_le_pow_of_one_le' := Left.one_le_pow_of_le
```

Comment:
Deprecation for this one seems to be missing; didn't check the others

Reply:
This shouldn't have been in the diff, sorry